modules/dr-restoring-cluster-state.adoc

```diff
@@ -209,7 +241,7 @@ $ sudo systemctl restart kubelet.service
 Clusters with no worker nodes, such as single-node clusters or clusters consisting of three schedulable control plane nodes, will not have any pending CSRs to approve. You can skip all the commands listed in this step.
 ====
 
-.. Get the list of current CSRs:
+.. Get the list of current CSRs, by running the following command:
```

Comment:
I think you can drop the comma as the sentence is short. With the comment, a subject is missing in the dependent clause.

Reply:
+1, no comma necessary